internal/transports/p2p/peer/peer.go

```diff
@@ -134,7 +130,7 @@ func (p *Peer) StartHeadersSync() error {
 	go p.readMsgHandler()
 
 	currentTipHeight := p.headersService.GetTipHeight()
-	p.setNextHeaderCheckpoint(currentTipHeight)
+	p.checkpoint.UseTip(currentTipHeight)
```

Comment:
We can just use `next` method directly here. We would be passing `currentTipHeight` to this method anyways, so it's descriptive enough IMO

Reply:
The possible solution in the comment above ☝️